boot/modeenv.go

```diff
@@ -111,6 +121,34 @@ func ReadModeenv(rootdir string) (*Modeenv, error) {
 	unmarshalModeenvValueFromCfg(cfg, "current_trusted_boot_assets", &m.CurrentTrustedBootAssets)
 	unmarshalModeenvValueFromCfg(cfg, "current_trusted_recovery_boot_assets", &m.CurrentTrustedRecoveryBootAssets)
 
+	// save all the rest of the keys we don't understand
+	keys, err := cfg.Options("")
+	if err != nil {
+		return nil, err
+	}
+	var knownKeys = []string{
```

Comment:
We should consider using struct tags for relevant fields at some point if we add more fields. This way it would be easier to keep track of known fields.

Reply:
that shouldn't be too hard, I'll give it a try